### ISSUE TITLE
Exp15: NVP_4 vs NVP_5

### ISSUE THREAD

**Issue by Micky774** [quinngroup/CiliaRepresentation#40]
While we are still trying to compare NVP_4 to our base NVP, it is also worth taking a look at NVP_5 (#39) which forfeits the texture resolution phase of the network. Similar to the configuration in Exp14 (#35) we will run NVP_5 under the same hyper-parameters. Tensorboard event history will also be copied from the results of #35 in order to more easily compare as training progresses.

**Comment by Micky774** [comment]
Trial in progress

**Comment by Micky774** [comment]
Trial completed and NVP_4 easily defeats NVP_5. With that being said, it's worth noting that NVP_4 has ~1.1mil more parameters (in the encoder) than NVP_5. Further study should be done on whether these parameters are better spent in a resnet configuration or in the texture resolution configuration that NVP_4 currently employs. 

It's also worth considering whether the decreased parameter count of NVP_5 would call for a decreased learning rate as well, and that consequently the learning rate was set too high leading to poor performance. Both should be tested.
